pkg/workflows/wasm/host/time_test.go

```diff
@@ -0,0 +1,87 @@
+package host
+
+import (
+	"context"
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk/v2/pb"
+)
+
+func TestTimeFetcher_GetTime_NODE(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	mockExec := NewMockExecutionHelper(t)
+	expected := time.Now()
+	mockExec.EXPECT().GetNodeTime().Return(expected)
+
+	tf := newTimeFetcher(ctx, mockExec)
+	tf.Start()
```

Comment:
fetcher should create a new child context it can cancel to expose a Close method